node/src/components/consensus/era_supervisor.rs

```diff
@@ -674,3 +688,43 @@ where
         }
     }
 }
+
+/// Network metrics to track Consensus
+#[derive(Debug)]
+pub struct ConsensusMetrics {
```

Comment:
Maybe this could move to its own file (`components/consensus/metrics.rs`). This module is already pretty large (and complicated).

Reply:
Can do!